mesheryctl/tests/e2e/001-system/01-system-context.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+setup() {
+   load "$E2E_HELPERS_PATH/tests_helpers"
+	_tests_helper
+
+    ENDPOINT_REGEX_MATCH='^[[:space:]]*endpoint:[[:space:]](http|https)://.*:[[:digit:]]+$'
+    TOKEN_REGEX_MATCH='^[[:space:]]*token:[[:space:]][[:alnum:]]+$'
+    PLATFORM_REGEX_MATCH='^[[:space:]]*platform:[[:space:]][kubernetes|docker]+$'
+    PROVIDER_REGEX_MATCH='^[[:space:]]*provider:[[:space:]][a-zA-Z]+$'
```

Comment:
curious if we mean to put kubernetes and docker as an alternation vs character class. https://www.regular-expressions.info/alternation.html

`[kubernetes|docker] vs (kubernetes|docker)`